domains/pallets/operator-rewards/src/lib.rs

```diff
@@ -61,14 +80,93 @@ mod pallet {
     #[pallet::hooks]
     impl<T: Config> Hooks<BlockNumberFor<T>> for Pallet<T> {
         fn on_initialize(_now: BlockNumberFor<T>) -> Weight {
-            BlockRewards::<T>::set(Zero::zero());
+            BlockRewards::<T>::set(Default::default());
```

Comment:
taking the value is more appropriate here than setting it default since `get` would any give you default due to `ValueQuery`